src/transformStateWithClones.js

```diff
@@ -7,7 +7,47 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+
+  let curentState = { ...state };
```

Comment:
There is a typo in the variable name 'curentState'. It should be corrected to 'currentState' to improve readability.